### PR TITLE
Move "Introduction to Mix" guide here

### PR DESCRIPTION
**Pull request by whatyouhide**

### CODE REVIEW

File: lib/mix/pages/introduction-to-mix.md [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+
+In order to build our key-value application, we are going to use three main tools:
+
+* ***OTP*** _(Open Telecom Platform)_ is a set of libraries that ships with Erlang. Erlang developers use OTP to build robust, fault-tolerant applications. In this chapter we will explore how many aspects from OTP integrate with Elixir, including supervision trees, event managers and more;
+
+* ***[Mix](`Mix`)*** is a build tool that ships with Elixir that provides tasks for creating, compiling, testing your application, managing its dependencies and much more;
+
+* ***[ExUnit](`ExUnit`)*** is a test-unit based framework that ships with Elixir.
+
+In this chapter, we will create our first project using Mix and explore different features in <abbr title="Open Telecom Platform">OTP</abbr>, Mix, and ExUnit as we go.
```
Comment (whatyouhide) [inline lib/mix/pages/introduction-to-mix.md:38]: ExDoc renders `abbr` by escaping the HTML. Suggestions on how to use `abbr` itself? 😊

Comment (josevalim) [reply lib/mix/pages/introduction-to-mix.md:38]: Oh, good find. Earmark does not support inline HTML and I used `<abbr>` in other places. Can you please make sure they are all removed?

File: lib/mix/pages/introduction-to-mix.md [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+
+<b>Fuck</b>
```
Comment (josevalim) [inline lib/mix/pages/introduction-to-mix.md:40]: image

File: lib/mix/pages/introduction-to-mix.md
Comment (josevalim) [inline lib/mix/pages/introduction-to-mix.md:1]: I thought we agreed we will ship this as part of Elixir itself and not Mix? There is one benefit in doing so: we will be able to cross-link from the other guides.

File: lib/mix/pages/introduction-to-mix.md (hunk 2 above)
Comment (whatyouhide) [reply lib/mix/pages/introduction-to-mix.md:40]: Ahahaha holy crap, I kept this in here to check inline HTML 😄

**Comment by whatyouhide** [comment]
@josevalim I had forgot :) Moved this to Elixir, removed `abbr`, and we're good to go.

File: lib/elixir/scripts/elixir_docs.exs [hunk 3]
```diff
@@ -56,7 +57,8 @@ canonical = System.fetch_env!("CANONICAL")
     "Getting started": ~r"pages/getting-started/.*\.md$",
     "Anti-patterns": ~r"pages/anti-patterns/.*\.md$",
     References: ~r"pages/references/.*\.md$",
-    "Meta-programming": ~r"pages/meta-programming/.*\.md$"
+    "Meta-programming": ~r"pages/meta-programming/.*\.md$",
+    Mix: ~r"pages/mix/.*\.md$"
```
Comment (josevalim) [inline lib/elixir/scripts/elixir_docs.exs:61]: I believe I was referring to those pages as /mix-and-otp/. there should be existing links and warnings when you running `make docs_elixir`.

**Review by josevalim — APPROVED** [review]
Just one comment on the path and we can ship it!

**Pull request merged**